staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -164,3 +189,15 @@ func (t *prefixTransformers) TransformToStorage(ctx context.Context, data []byte
 	prefixedData = append(prefixedData, result...)
 	return prefixedData, nil
 }
+
+// getDataPrefix returns the encryption prefix
+// Format: k8s:enc:<encryption-type>:<encryption-version>:
+// For example: k8s:enc:kms:v2:
+func getDataPrefix(data []byte) string {
```

Comment:
This is incorrect, `prefixTransformers` need not be used at only the root, for example see the code in `aesPrefixTransformer`.